src/renderer/global.scss

```diff
@@ -1,4 +1,5 @@
 @use "colors.scss";
+@import "reactflow/dist/style.css";
```

Comment:
Does this even work? Can you use npm-style imports in SCSS files?

Reply:
Yes. Not only does it work, it's necessary.